iFixit/react-commerce
# Fix cart drawer animations

closes #1930 

I took this bug as an opportunity to fix other cart drawer animations bugs as well. 

Given that the problem seems to be a bug in framer motions components I replaced the animation with CSS animations. This change down the road will help us get rid of framer motion ([~40kb min/gzip](https://bundlephobia.com/package/framer-motion@10.16.2)).

## QA

Open [Vercel preview](https://react-commerce-git-fix-crat-drawer-animation-bug-ifixit.vercel.app/products/pro-tech-toolkit) and verify that bug described by #1930 is fixed
